gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -167,6 +168,24 @@ public void commit()
     if (!this.fs.rename(this.stagingFile, this.outputFile)) {
       throw new IOException("Failed to commit data from " + this.stagingFile + " to " + this.outputFile);
     }
+
```

Comment:
extracted a method. But I'm keeping it in the same class since it is very context specific. Do we have another use case in which this method may be useful?